lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -4491,3 +4616,98 @@ LogicalResult FIRRTLLowering::visitStmt(ProbeOp op) {
 
   return success();
 }
+
+LogicalResult FIRRTLLowering::fixupLTLOps() {
```

Comment:
If the worklist was a SetVector, much of this might disappear.

Reply:
Oh sweet, I was under the impression that `SetVector` didn't support removal. I was very wrong! Thanks for the pointer.